### DISCUSSION TITLE
legacy_directory_do_not_use.txt ?

### DISCUSSION THREAD

**Discussion by EdnaMode22** [cubiq/ComfyUI_IPAdapter_plus#730] [Q&A]
IN the Models folder there is this text file and any model I put in this folder would not show up in the ComfyUI, did I do something wrong installing? 
![Screenshot 2024-10-04 221723](https://github.com/user-attachments/assets/89ce11cb-965c-4a7a-9dd5-eacc0bdba840)
![Screenshot 2024-10-04 221730](https://github.com/user-attachments/assets/41116d95-0bcd-4881-88b6-27d864c71318)

**Comment by neverbiasu** [comment]
ipadapter models should be placed in `ComfyUI/models/ipadapter`
